### PR TITLE
docs(collaborator_guide): add collaborator guide

### PR DESCRIPTION
**Pull request by gkatsev**
This adds a collaborator guide that describes how to reply to issues and
PRs and how to accept changes to the project.

### CODE REVIEW

**Comment by gkatsev** [comment]
@dmlap @heff @mmcc please review.

File: COLLABORATOR_GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+
+Before a PR is merged, it must be reviewed by at least two core committers, at least one if it comes from a core committer.
+
+Feel free to @-mention a particular core committer if you know they are experts in the area that is being changed.
+
+If you are unsure about the modification and cannot take responsibility for it, defer to another core committer.
+
+Before merging the change, it should be left open for other core committers to comment on. At least 24 hours during a weekday, and the 48 hours on a weekend. Trivial changes or bug fixes that have been reviewed by multiple committers may be merged without delay.
+
+For non-breaking changes, if there is no disagreeming between the collaborators, the PR may be landed assuming it was reviewed. If there is still disagreement, it may need to be [escalated to the TSC](#involving-the-tsc).
```
Comment (heff) [inline COLLABORATOR_GUIDE.md]: I disagreem with everything!

File: COLLABORATOR_GUIDE.md [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Issues and Pull Requests
+
+Full courtesey should always be shown in video.js projects.
+
+Collaborators may manage issues they feel qualified to handle, being mindful of our guidelines.
+
+Any issue and PR can be closed if they are not relevant, when in doubt leave it open for more discussion. Issues and always be re-opened if new information is made available.
```
Comment (heff) [inline COLLABORATOR_GUIDE.md]: Issues and always be

File: COLLABORATOR_GUIDE.md [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+* The change is inherently controversial
+* No agreement was reached between collaborators participating in the discussion
+
+The TSC will be the final arbiter when required.
+
+## Landing a PR
+
+Landing a PR is fairly easy given that we can use the Github UI for it.
+
+When using the big green button on github, make sure the "squash and merge" is selected -- it should be the only allowed option. If a PR has two features in it and should be merged as two separate commits, either ask the contributor to break it up into two, or follow the [manual steps](#langing-a-pr-manually).
```
Comment (mister-ben) [inline COLLABORATOR_GUIDE.md]: langing

File: COLLABORATOR_GUIDE.md [hunk 4]
```diff
@@ -0,0 +1,318 @@
...
+
+The commit message should follow our [conventional changelog conventions][conventions]. They are based on the angularjs changelog conventions. The changelog is then generated from these commit messages on release.
+
+The first line of the commit message -- the header and first text box on github -- should be prefixed with a type and optional scope followed by a short description of the commit.
```
Comment (mister-ben) [inline COLLABORATOR_GUIDE.md]: Minor niggle, but the capitalisation of 'Github' is inconsistent

**Pull request merged**